packages/blade/src/components/PreviewWindow/docs/PreviewWindow.stories.tsx

```diff
@@ -0,0 +1,333 @@
+import type { StoryFn, Meta } from '@storybook/react';
+import React, { useState } from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { Document, Page as ReactPdfPage, pdfjs } from 'react-pdf';
+import { PreviewBody, PreviewHeader, PreviewWindow, PreviewFooter } from '../PreviewWindow';
+import type { PreviewWindowProps } from '../types';
+import { Heading, Text } from '~components/Typography';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { Card, CardBody, CardHeader, CardHeaderLeading } from '~components/Card';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'react-pdf/dist/esm/Page/AnnotationLayer.css';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import 'react-pdf/dist/esm/Page/TextLayer.css';
+import { ArrowLeftIcon, ArrowRightIcon } from '~components/Icons';
+
+// Set up the worker
+pdfjs.GlobalWorkerOptions.workerSrc = new URL(
+  'pdfjs-dist/build/pdf.worker.min.mjs',
+  import.meta.url,
+).toString();
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="PreviewWindow"
+      componentDescription="A Preview Window is a component that is used to preivew a file"
+      apiDecisionLink={null}
+      figmaURL="https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=100413-32686&t=n9A7LztwEkIsly3v-0"
+    >
+      <Heading size="large">Usage</Heading>
+      <Sandbox showConsole>
+        {`
+        import { PreviewBody, PreviewHeader, PreviewWindow, PreviewFooter } from '@razorpay/blade/components';
+        
+        function App() {
+          return (
+             <PreviewWindow>
+             <PreviewHeader title="Preview " />
+              <PreviewBody>
+                <img width="100%" height="100%" src="https://picsum.photos/1920/1080" alt="randomImage" />
+              </PreviewBody>
+            <PreviewFooter showZoomPercentage={true} />
+            </PreviewWindow>
+          )
+        }
+
+        export default App;
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/PreviewWindow',
+  component: PreviewWindow,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<PreviewWindowProps>;
+
+const PreviewWindowTemplate: StoryFn<typeof PreviewWindow> = () => {
+  return (
+    <PreviewWindow>
```

Comment:
Should the component just be called `<Preview>`? `PreviewWindow` sounds like its some nested component unless we name inner components `PreviewWindowHeader` etc.

Preview sounds fine I guess?